lib/gui/app/components/wifi/react/actions.js

```diff
@@ -21,7 +21,7 @@ const url = require('url')
 
 const wifiServiceEndpoint = url.format({
   protocol: 'http',
-  hostname: process.env.ETCHER_WIFI_SERVICE_IP || 'localhost',
+  hostname: process.env.ETCHER_WIFI_SERVICE_IP || '10.211.55.3' || 'localhost',
```

Comment:
I don't think that IP is supposed to be in here ;)

Reply:
Definetly not. Fixed